DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: berdie
 Type: Package
 Title: Berdie
-Version: 0.0.2
+Version: 0.0.3
```

Comment:
It's a pretty large change to the package.  Also, you should feel good; I've found it difficult to integrate Postgres libraries in the past.